tests/integration/household_composition/test_repeating_household.py

```diff
@@ -92,6 +96,46 @@ def test_change_repeat_answer_removes_all_answers_for_repeated_groups(self):
         self.assertRegex(content, 'What is their age')
         self.assertNotRegex(content, '9992')
 
+    def test_no_change_repeat_answer_keeps_repeated_groups(self):
+
+        # Given I add some people
+        household_composition_page = self.first_page
+        form_data = MultiDict()
```

Comment:
If you named this household_form_data and the other age_form_data we don't need to duplicate the form data and it's more obvious we are posting the same format data